.github/workflows/dotnet.yml

```diff
@@ -41,11 +41,17 @@ jobs:
       run: dotnet build --no-restore
 
     - name: Run unit tests
-      run: dotnet test BTCPayServer.Plugins.Monero.UnitTests --verbosity normal
+      run: dotnet test BTCPayServer.Plugins.Monero.UnitTests/BTCPayServer.Plugins.UnitTests.csproj /p:CollectCoverage=true /p:CoverletOutputFormat=cobertura /p:CoverletOutput=coverage/coverage.cobertura.xml
 
     - name: Run integration tests
       run: docker compose -f BTCPayServer.Plugins.IntegrationTests/docker-compose.yml run tests
 
     - name: Cleanup Docker containers
       if: always()
-      run: docker compose -f BTCPayServer.Plugins.IntegrationTests/docker-compose.yml down -v
\ No newline at end of file
+      run: docker compose -f BTCPayServer.Plugins.IntegrationTests/docker-compose.yml down -v
+
+    - name: Upload integration test coverage
```

Comment:
you should upload this artifact to Codacy i believe.. most probably with https://github.com/codacy/codacy-coverage-reporter-action.. in a separated workflow.. conditioned to this repo.. check [haveno codacy](https://github.com/haveno-dex/haveno/blob/master/.github/workflows/codacy-code-reporter.yml) or [bisq codacy](https://github.com/bisq-network/bisq/blob/master/.github/workflows/codacy-coverage-reporter.yml)

Reply:
ah.. sorry.. commenting outdated parts here lol.. still.. u should condition so if someone fork it wont fail.. maybe a separated workflow ?